docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -18,14 +18,17 @@ image::images/dataview-button-highlighted.png[image highlighting how to open the
 [discrete]
 == Create or modify a {data-source}
 
-You can temporarily modify the active {data-source} from the *{data-source-cap}* menu by clicking *Advanced options*, then adding or removing index patterns.
+To learn how to permanently modify the default "Security Data View", refer to <<update-sec-indices, Update default {elastic-sec} indices>>.
+
+To learn how to permanently modify, create, or delete another {data-source} refer to {apm-app-ref}/data-views.html[{kib} {data-sources-cap}].
+
+You can also temporarily modify the active {data-source} from the *{data-source-cap}* menu by clicking *Advanced options*, then adding or removing index patterns.
 
 image::images/dataview-filter-example.gif[video showing how to filter the active data view]
 
 This only allows you to add index patterns that match indices that currently contain data (other index patterns are unavailable). Note that any changes made are saved in the current browser window and won't persist if you open a new tab.
 
-To permanently modify a {data-source}, delete an existing {data-source} or create a new one, you need the required permissions.
-To learn more, refer to {apm-app-ref}/data-views.html[{kib} {data-sources-cap}].
+NOTE: You cannot update the data view for the Alerts page. It always shows data from `.alerts-security.alerts-default`.
```

Comment:
@e40pud can we tweak the UI copy so it aligns with this note? The current copy could be interpreted as: you must go elsewhere to update the data view for the Alerts page. 

<img width="400" alt="Screenshot 2024-01-30 at 2 55 42 PM" src="https://github.com/elastic/security-docs/assets/79124755/17551b18-f18e-43b6-8732-79b449946f66">


If it's still possible to change the copy, I'd suggest this:

You can't modify the data view for the Alerts page. 

If that version is too blunt and overly formal, here's a slightly softer version:

This page's data view can't be adjusted.

Reply:
It is not bound to Alerts page only. Same happens on rule details page. Also, there is another similar message that we show on the timeline. So, general message looks better to me.

https://github.com/elastic/security-docs/assets/2700761/f47b0b78-9939-41e0-9691-c097fb695076

Reply:
Ah, interesting. Thanks for pointing out these other areas @e40pud -- I wasn't aware of them. Would it be ok if we continued this conversation in https://github.com/elastic/security-docs/issues/4718 so that we're not blocking this doc issue from being closed and merged?